### PR TITLE
Semantic HTML for beginners

### LINKED ISSUE #2603: Semantic HTML for Beginners
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of article
Semantic HTML for Beginners 

### Introduction paragraph (2-3 paragraphs):
Writing HTML without semantics is not recommended for efficient code, it can definitely work with little to no issue, but there is rather a better way of structuring our HTML. This is known as semantic HTML. 

This approach gives meaning to our HTML structure and it can be very useful to get other frontend developers to quickly understand the codebase.

In this tutorial, you will be learning how to write readable HTML code.

### Key takeaways: 
Why semantics HTML
Advantages of Semantic HTML
Different examples of semantic HTML

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by josax01**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
This closes #2603
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [ ] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [ ] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by josax01** [comment]
This fixes #2603

**Comment by lalith1403** [comment]
Hey @josax0, is this PR reviewed?

**Comment by josax01** [comment]
Hi, @lalith1403, no it isn't

**Comment by josax01** [comment]
No,it isn't.

On Jul 13, 2021 16:45, "Lalithnarayan C" ***@***.***> wrote:

Hey @josax0, is this PR reviewed?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/section-engineering-education/engineering-education/pull/2872#issuecomment-879199358>,
or unsubscribe

**Comment by BrianaNzivu** [comment]
> Hey @josax0, is this PR reviewed?

Hey @lalith1403 .👋🏾
Were you planning to review the article?

**Comment by lalith1403** [comment]
Please proceed, @BrianaNzivu.

File: content/articles/semantic-html-for-beginners/index.md [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+1. we have the middle area which is called the `main` tag in HTML5.
+1. we have the `footer` tag showcasing the closing section of the web page.
+1. in the `main` area we can have a lot of things inside. Semantic HTML comes baked with its special tag some of them include `section`, `article`, and `aside` tags.
+1. the `section` area showcase different sections of our web page. Multiple sections can be created. the `section` can be in the `header`, `footer`.
+1. the `aside` area showcase the area of our web page not too relevant. It can also serve as the sidebar.
+
+Finally, these are the basic semantic HTML5 tags, which can make our HTML more readable and easier to understand. However, it does not affect the view of the web page.
+
+### Conclusion
+
```
Comment (BrianaNzivu) [inline content/articles/semantic-html-for-beginners/index.md:396]: The conclusion is a recap of the article.
So give a short summary of what your article has covered (Keep it two to three sentences)

**Review by BrianaNzivu — CHANGES_REQUESTED** [review]
Hey Joshua :wave: .
First things first welcome to Section Eng-Ed. :tada: 
We are happy to have you here.
My name is Briana and I will be taking you through the review process to make your article almost perfect if not perfect for publishing.

Your article is a very interesting read. 
Good job!
I have done the first review and I have requested a few changes.

I have also noticed that your hero image is not 600 x 400. Please try and resize the image as explained in the instructions in the checklist.
 Since your article is more of a how-to guide, please read this [document](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md) and structure your article like this.

Please ping me when you are done with the corrections.

In case of any problem, question or clarification do not hesitate to reach out. I am here to help.

Regards.

**Comment by josax01** [comment]
@BrianaNzivu thanks for the review, I have made the needed changes 👍

**Comment by josax01** [comment]
@BrianaNzivu any update :]

**Comment by BrianaNzivu** [comment]
Hey @josax01 👋🏾.
I have been wrapping up on another article but I will proceed with yours probably tomorrow.
Incase of any updates, I will let you know.

Regards 🚀

**Comment by josax01** [comment]
@BrianaNzivu any update on your review

**Comment by hectorkambow** [comment]
plagiarism check done

**Review by BrianaNzivu — APPROVED** [review]
Hey @josax01 :wave: .
First, I'd like to apologize for the delay. 
A personal emergency came up but now I am back.

Next, Good job on the article!
An interesting read.

I'd like to point out a few things which you can implement in your next articles:
1. It is always advisable to submit an article that is well polished. By this, I mean having an article that does not have too many grammatical corrections and functional code. I suggest that you check out our [resources](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) page which has links to various resourceful tools for writing.
2. I always know it is a good practice to have your CSS file separate from the HTML file. This time I won't raise any red flags since the focus of the article is not on styling.
3.  Just like how it is advisable to have indented code when coding, It is also advisable to have well-indented code snippets for your article.
4. Lastly please go through all the corrections I have made so that you can not repeat the same mistakes. Remember your first article is normally for you to learn.

That's it! :smile: 

Next, a content moderator will do a final review. After that, your article will be published.

Otherwise,
Congratulations on publishing your first article!! 🥳 

In case of any question or clarification (currently or in the future), do not hesitate to reach out via [Slack](https://app.slack.com/client/T5X14SH7G/D01A6MT673R) or my personal email (briananzivu@gmail.com). 
I am **ALWAYS** here to help.

It has been a pleasure reading and reviewing this article.

Looking forward to seeing you contribute more to EngEd. 🚀

Kind Regards.

**Comment by LinusMuema** [comment]
Hi @josax01 
I see you were given a proper welcome by your reviewer. Thanks for submitting your first article and we hope to see more of your great content. You can check the review changes for future articles.

cc: @BrianaNzivu :rocket:

**Pull request merged**